sonic_y_cable/y_cable.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#
+# y_cable.py
+#
+#  class for implementing Y cable access and configurations
+#   API's for Y cable functionality in SONiC
+
+try:
+    import binascii
+    import sys
+
+    import sonic_platform.platform
+    import sonic_platform_base.sonic_sfp.sfputilhelper
+
+    from sonic_py_common import logger
+
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+
+PLATFORM_SPECIFIC_MODULE_NAME = "sfputil"
+PLATFORM_SPECIFIC_CLASS_NAME = "SfpUtil"
+
+# definitions of the offset and width for values of MUX register specs of upper page 0x04 info eeprom
+
+Y_CABLE_DETERMINE_CABLE_READ_SIDE = 640
+Y_CABLE_CHECK_LINK_ACTIVE = 641
+Y_CABLE_SWITCH_MUX_DIRECTION = 642
+Y_CABLE_ACTIVE_TOR_INDICATOR = 645
+
+
+class Y_cable(object):
```

Comment:
removed the class name, Since we don't need to instantiate these objects and maintain a state
I think we can just import the package and directly call these API's
Thoughts ?